ahcore/writers.py

```diff
@@ -170,6 +175,8 @@ def init_writer(self, first_coordinates: GenericArray, first_batch: GenericArray
             "format": _format,
             "dtype": str(first_batch.dtype),
             "is_binary": self._is_compressed_image,
+            "precision": self._precision.value if self._precision else "FP32",
+            "multiplier": self._precision.get_multiplier() if self._precision else 1.0,
```

Comment:
Similar here, probably best to use the values from the actual Type you created, such that any changes in the future will be captured by changing the the type, instead of having to scout out these values as well.